shared/elements/gaia_grid/js/grid_layout.js

```diff
@@ -171,7 +171,6 @@
             this.cols = parseInt(prop.value, 10);
             this.gridView.render();
           }
-
```

Comment:
Unnecessary change - I'm not against it, but better to change this when other changes are being made in this file than randomly here.